x-pack/legacy/plugins/siem/public/containers/detection_engine/rules/types.ts

```diff
@@ -78,8 +78,12 @@ export const RuleSchema = t.intersection([
     updated_by: t.string,
   }),
   t.partial({
+    last_failure_at: t.string,
+    last_failure_message: t.string,
     output_index: t.string,
```

Comment:
Should `last_success_at` and `last_success_message` be included in this? Does not have to be, but just wondering.